src/plugins/spark/features/ActuatorPwm/ActuatorPwmWidget.vue

```diff
@@ -50,6 +55,12 @@ export default class ActuatorPwmWidget extends BlockWidget {
       <q-btn slot="right" flat round dense icon="refresh" @click="refreshBlock"/>
     </q-card-title>
     <q-card-separator/>
+    <q-alert
+      v-if="!block.data.enabled"
+      :actions="[{label:'Enable', handler: enable }]"
+      type="info"
+      color="info"
+    >Toggling {{ block.data.actuatorId }} is disabled</q-alert>
```

Comment:
change to active speech. Avoid confusion with the non-jargon meaning of "driving".

Reply:
suggestion:
"This block is disabled. actuatorId can't be toggled."